Shared/Samples/Style features with custom dictionary/README.md

```diff
@@ -0,0 +1,44 @@
+# Style features with custom dictionary
+
+Use a custom dictionary style created from a web style or local style file (.stylx) to symbolize features using a variety of attribute values.
+
+![Image of style features with custom dictionary sample](style-features-with-custom-dictionary.png)
+
+## Use case
+
+When symbolizing geoelements in your map, you may need to convey several pieces of information with a single symbol. You could try to symbolize such data using a unique value renderer, but as the number of fields and values increases, that approach becomes impractical. With a dictionary renderer you can build each symbol on-the-fly, driven by one or more attribute values, and handle a nearly infinite number of unique combinations.
+
+## How to use the sample
+
+Toggle between the dictionary symbols from the web style and style file. Pan and zoom around the map to see the symbology from the chosen dictionary symbol style. The web style and style file are slightly different to each other to give a visual indication of the switch between the two.
+
+## How it works
+
+1. Create a `PortalItem`, referring to a `Portal` and the item ID of the web style.
+2. Based on the style selected:
+*  If the web style toggle has been selected, create a new `DictionarySymbolStyle` from the portal item using `DictionarySymbolStyle(portalItem:)`, and load it.
+*  If the file style toggle has been selected, create a new `DictionarySymbolStyle` using `DictionarySymbolStyle.createFromFile(url:)`.
```

Comment:
In the Swift API, it is not `DictionarySymbolStyle.createFromFile(url:)`, it is [DictionarySymbolStyle(url:)](https://developers.arcgis.com/swift/api-reference/documentation/arcgis/dictionarysymbolstyle/init(url:)).